eventmesh-dashboard-common/src/main/java/org/apache/eventmesh/dashboard/common/properties/RocketmqProperties.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.dashboard.common.properties;
+
+import lombok.Data;
+
+@Data
+public class RocketmqProperties {
+
+    private String namesrvAddr;
+
+    private String clusterName;
+
+    private String brokerUrl;
+
+    private String endPoint;
+
+    private int writeQueueNums = 8;
+
+    private int readQueueNums = 8;
```

Comment:
No need to perform queue scaling, so you don't have to set `readQueueNums` and `writeQueueNums`.

Reply:
ok